### ISSUE TITLE
Exception traceback bug with keyword arguments

### ISSUE THREAD

**Issue by AnesBenmerzoug** [Delgan/loguru#144]
I noticed a confusing bug when variable values are shown in tracebacks.

This bug can reproduced using the following snippet:

```python
from loguru import logger

def f(x):
    return 1 / x

if __name__ == "__main__":
    y = 0
    with logger.catch():
        f(x=y)

    x = 0
    with logger.catch():
        f(x=x)
```

We notice that in the second traceback that when the named argument and the passed variable share the same name they're both treated as being variables and their values are shown.
Whereas in the first one, only the variable's value is shown.

```python
2019-09-24 21:04:17.405 | ERROR    | __main__:<module>:11 - An error has been caught in function '<module>', process 'MainProcess' (23269), thread 'MainThread' (140248716965696):
Traceback (most recent call last):

> File "test.py", line 11, in <module>
    f(x=y)
    │   └ 0
    └ <function f at 0x7f8e32de7378>

  File "test.py", line 5, in f
    return 1 / x
               └ 0

ZeroDivisionError: division by zero
2019-09-24 21:04:17.416 | ERROR    | __main__:<module>:15 - An error has been caught in function '<module>', process 'MainProcess' (23269), thread 'MainThread' (140248716965696):
Traceback (most recent call last):

> File "test.py", line 15, in <module>
    f(x=x)
    │ │ └ 0
    │ └ 0
    └ <function f at 0x7f8e32de7378>

  File "test.py", line 5, in f
    return 1 / x
               └ 0

ZeroDivisionError: division by zero
```

**Comment by Delgan** [comment]
Thanks for the bug report. ;)

I'm not sure this can be easily fixed. The parser implemented in Loguru to display such traceback is very limited. I probably could ignore variable assignment with surrounding parentheses, though.

**Comment by AnesBenmerzoug** [comment]
It's not really catastrophic. I just wanted to report it before I forgot about it.
If it would take too much work to fix, then I think it's better to invest that time and effort into something more useful.

**Comment by Delgan** [comment]
Fixed! It will be available in the next `v0.4.0` release. Thanks again for the report. :wink:
